class1.html

```diff
@@ -83,6 +82,21 @@
         </section>
       </section>
 
+        <section>
+          <h3>Scary things we'll be facing</h3>
```

Comment:
This is a chance to pause and emphasize all the ways coding class can be hard (so be sure to ask questions)